src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -72,7 +72,7 @@ export class StartCaseBreadcrumbsResolver extends BreadcrumbsResolver {
   resolve(route: ActivatedRouteSnapshot) {
     const path = route.params.name;
     const text = _.startCase(path);
-    return [{ text: `${text}/Edit`, path: path }];
+    return [{ text: `${text}/Edit`, path }];
```

Comment:
We already discussed this in the original PR... While this is [perfectly valid ES6 syntactic sugar](https://eslint.org/docs/rules/object-shorthand), I think it's an ugly one, that may lead to future bugs as it creates tight coupling between internal variable names and external interfaces (e.g: if someone refactors this code and renames `path` to `pathname`, the resulting object will have a new interface). I'd vote for disabling this rule.